docs/tutorials/basics.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "7FV_m2u2Hny8"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    not_op = not_gate(q0, q1)
</pre><p>cnot_op instead of not_op</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>